### PR TITLE
Feat/text change

### PR DESCRIPTION
**Pull request by kallepronk**

### CODE REVIEW

**Review by xl3ehindTim — APPROVED** [review]
Lgtm!

File: src/index.css [hunk 1]
```diff
@@ -109,9 +109,9 @@ body {
 
 
 .pulsing-gradient {
-    background: linear-gradient(0deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+    background: linear-gradient(0deg, #8e4731, #912650, #186d8c, #188d72, #8e4731, #912650, #186d8c, #188d72, #8e4731, #912650, #186d8c, #188d72, #8e4731, #912650, #186d8c, #188d72);
```
Comment (BartvanEijkelenburg) [inline src/index.css:112]: The brown looks really dirty, mind changing that color of the gradient?

**Review by BartvanEijkelenburg — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 1 above)
Comment (kallepronk) [reply src/index.css]: changed

**Review by BartvanEijkelenburg — APPROVED** [review]
Woop, Woop, Nice change

**Pull request merged**